src/features/newCommentFormSlice.tsx

```diff
@@ -0,0 +1,39 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = {
+  submitting: false,
+  newCommentFormErrors: {
+    name: false,
+    body: false,
+    email: false,
+  },
+  name: '' as string,
+  email: '' as string,
+  body: '' as string,
```

Comment:
try to avoid using `as`
```suggestion
  name: '',
  email: '',
  body: '',
```